front/src/common/RollingStockSelector/RollingStock2Img.tsx

```diff
@@ -27,19 +30,20 @@ const RollingStock2Img: React.FC<{ rollingStock: RollingStock | LightRollingStoc
     }
   };
 
+  const displayImage = () => (
+    <LazyLoadImage src={imageUrl || ''} alt={rollingStock?.name || 'defaultImg'} />
+  );
+
   useEffect(() => {
     setImageUrl('');
     getRollingStockImage();
   }, [rollingStock]);
 
-  return imageUrl ? (
-    <LazyLoadImage src={imageUrl} alt={rollingStock?.name} />
-  ) : (
-    // TODO : replace with null when images RTK query is ready in RollingStockEditor
-    <div className="defaultImageRSEditor">
-      <img src={blueTrain} alt="defaultImage" title="tchou tchou" />
-    </div>
-  );
+  useEffect(() => {
+    displayImage();
+  }, [imageUrl]);
+
+  return displayImage();
```

Comment:
When a state variable changes, the component is automatically re-rendered ! So each time imageUrl will change, the LazyLoadImage will be re-rendered
```suggestion
  return <LazyLoadImage src={imageUrl || ''} alt={rollingStock?.name || 'defaultImg'} />;
```

Reply:
Done :)